### PR TITLE
client/server: misc comments

### PR DESCRIPTION
**Pull request by tmakatos**
Signed-off-by: Thanos Makatos <thanos.makatos@nutanix.com>

### CODE REVIEW

File: samples/server.c [hunk 1]
```diff
@@ -536,12 +536,23 @@ int main(int argc, char *argv[])
...
                 if (ret < 0) {
                     err(EXIT_FAILURE, "vfu_irq_message() failed");
                 }
 
+                /*
+                 * We also initiate some dummy DMA via an explicit message,
+                 * again to show how DMA is done. This is used if the client's
+                 * RAM isn't mappable or the server implementation prefers it
+                 * this way.  Again, the client expects the server to send DMA
+                 * messages right after is has triggerd the IRQs.
```
Comment (jlevon) [inline samples/server.c:554]: nit, "it has triggered"

**Review by jlevon — CHANGES_REQUESTED** [review]

File: samples/server.c (hunk 1 above)
Comment (tmakatos) [reply samples/server.c:554]: ok

File: samples/server.c [hunk 2]
```diff
@@ -536,12 +536,23 @@ int main(int argc, char *argv[])
...
                 if (ret < 0) {
                     err(EXIT_FAILURE, "vfu_irq_message() failed");
                 }
 
+                /*
+                 * We also initiate some dummy DMA via an explicit message,
+                 * again to show how DMA is done. This is used if the client's
+                 * RAM isn't mappable or the server implementation prefers it
+                 * this way.  Again, the client expects the server to send DMA
+                 * messages right after is has triggered the IRQs.
```
Comment (jlevon) [inline samples/server.c:554]: missed the "is->it" :)

**Review by jlevon — CHANGES_REQUESTED** [review]

File: samples/server.c (hunk 2 above)
Comment (tmakatos) [reply samples/server.c:554]: ok

File: samples/server.c [hunk 3]
```diff
@@ -536,12 +536,23 @@ int main(int argc, char *argv[])
...
                 if (ret < 0) {
                     err(EXIT_FAILURE, "vfu_irq_message() failed");
                 }
 
+                /*
+                 * We also initiate some dummy DMA via an explicit message,
+                 * again to show how DMA is done. This is used if the client's
+                 * RAM isn't mappable or the server implementation prefers it
+                 * this way.  Again, the client expects the server to send DMA
+                 * messages right after it is has triggered the IRQs.
```
Comment (jlevon) [inline samples/server.c:554]: one more round, s/it is/it/

**Review by jlevon — CHANGES_REQUESTED** [review]

File: samples/server.c (hunk 3 above)
Comment (tmakatos) [reply samples/server.c:554]: sorry, brain isn't working today

Comment (tmakatos) [reply samples/server.c:554]: if it takes 3 rounds to review a PR with only comments then how long will it take for the PRs with code changes ? xD

**Review by jlevon — APPROVED** [review]
Oh, I dunno, it's much harder to tell if actual code is correct or not :)

**Pull request merged**